### PR TITLE
docs: Add BTC Signet to readme and .env.example

### PR DESCRIPTION
**Pull request by duanyytop**
## Changes

- Add BTC Signet description to the readme and .env.example
- Use Signet as default for BTC network

### CODE REVIEW

**Review by ahonn — APPROVED** [review]

File: apps/service/.env.example [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+BTC_TESTNET_TYPE=Signet
 
-# CKB node url which should be matched with NETWORK
+# CKB node url which should match NETWORK
 CKB_RPC_URL=https://testnet.ckb.dev
 
-# The BTC assets api url which should be matched with NETWORK
-BTC_SERVICE_URL=https://btc-assets-api.testnet.mibao.pro
+# The BTC assets api url which should match NETWORK
+# The BTC Testnet Service url is: https://btc-assets-api.testnet.mibao.pro
```
Comment (Flouse) [inline apps/service/.env.example:13]:
```suggestion
# The BTC Testnet Service url is: https://api.testnet.rgbpp.io
```

Let's use new alias domain from now on.

File: apps/service/.env.example [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
+BTC_SERVICE_URL=https://api.signet.rgbpp.io
 
-# The BTC assets api token which should be matched with NETWORK and BTC_TESTNET_TYPE
+# The BTC assets api token which should match NETWORK and BTC_TESTNET_TYPE
 # To get an access token, please refer to https://github.com/ckb-cell/rgbpp-sdk/tree/develop/packages/service#get-an-access-token
 BTC_SERVICE_TOKEN=
 
-# The BTC assets api origin which should be matched with NETWORK and BTC_TESTNET_TYPE
-BTC_SERVICE_ORIGIN=https://btc-assets-api.testnet.mibao.pro
+# The BTC assets api origin which should match NETWORK and BTC_TESTNET_TYPE
```
Comment (Flouse) [inline apps/service/.env.example:21]:
```suggestion
# The BTC assets api origin which should match `BTC_SERVICE_TOKEN`
# JTS Debugger: https://jwt.io
```

File: examples/rgbpp/.env.example [hunk 3]
```diff
@@ -28,13 +28,15 @@ BTC_PRIVATE_KEY=private-key
 # Read more about P2WPKH in BIP141: https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki#p2wpkh
 BTC_ADDRESS_TYPE=P2WPKH
 
-# The BTC assets api url which should be matched with IS_MAINNET and BTC_TESTNET_TYPE
-VITE_BTC_SERVICE_URL=https://btc-assets-api.testnet.mibao.pro
+# The BTC assets api url which should match IS_MAINNET and BTC_TESTNET_TYPE
+# The BTC Testnet Service url is: https://btc-assets-api.testnet.mibao.pro
```
Comment (Flouse) [inline examples/rgbpp/.env.example:32]:
```suggestion
# The BTC Testnet Service url is: https://api.testnet.rgbpp.io
```

File: apps/service/README.md [hunk 4]
```diff
@@ -23,26 +23,28 @@ $ cp .env.example .env
...
+BTC_SERVICE_URL=hhttps://api.signet.rgbpp.io
 
-# The BTC assets api token which should be matched with NETWORK and BTC_TESTNET_TYPE
+# The BTC assets api token which should match NETWORK and BTC_TESTNET_TYPE
 # To get an access token, please refer to https://github.com/ckb-cell/rgbpp-sdk/tree/develop/packages/service#get-an-access-token
 BTC_SERVICE_TOKEN=
 
-# The BTC assets api origin which should be matched with NETWORK and BTC_TESTNET_TYPE
-BTC_SERVICE_ORIGIN=https://btc-assets-api.testnet.mibao.pro
+# The BTC assets api origin which should match NETWORK and BTC_TESTNET_TYPE
```
Comment (Flouse) [inline apps/service/README.md:46]:
```suggestion
# The BTC assets api origin which should match `BTC_SERVICE_TOKEN`
# JTS Debugger: https://jwt.io
```

File: apps/service/README.md [hunk 5]
```diff
@@ -23,26 +23,28 @@ $ cp .env.example .env
...
+BTC_TESTNET_TYPE=Signet
 
-# CKB node url which should be matched with NETWORK
+# CKB node url which should match NETWORK
 CKB_RPC_URL=https://testnet.ckb.dev
 
-# The BTC assets api url which should be matched with NETWORK
-BTC_SERVICE_URL=https://btc-assets-api.testnet.mibao.pro
+# The BTC assets api url which should match NETWORK and BTC_TESTNET_TYPE
+# The BTC Testnet Service url is: https://btc-assets-api.testnet.mibao.pro
```
Comment (Flouse) [inline apps/service/README.md:38]: ditto

**Review by Flouse — APPROVED** [review]

File: apps/service/.env.example [hunk 6]
```diff
@@ -1,20 +1,23 @@
...
 
-# The BTC assets api token which should be matched with NETWORK and BTC_TESTNET_TYPE
+# The BTC assets api token which should match NETWORK and BTC_TESTNET_TYPE
 # To get an access token, please refer to https://github.com/ckb-cell/rgbpp-sdk/tree/develop/packages/service#get-an-access-token
 BTC_SERVICE_TOKEN=
 
-# The BTC assets api origin which should be matched with NETWORK and BTC_TESTNET_TYPE
-BTC_SERVICE_ORIGIN=https://btc-assets-api.testnet.mibao.pro
+# The BTC assets api origin which should match `BTC_SERVICE_TOKEN`
+# JTS Debugger: https://jwt.io
```
Comment (Flouse) [inline apps/service/.env.example:22]:
```suggestion
# JWT Debugger: https://jwt.io
```

File: apps/service/README.md [hunk 7]
```diff
@@ -23,26 +23,29 @@ $ cp .env.example .env
...
 
-# The BTC assets api token which should be matched with NETWORK and BTC_TESTNET_TYPE
+# The BTC assets api token which should match NETWORK and BTC_TESTNET_TYPE
 # To get an access token, please refer to https://github.com/ckb-cell/rgbpp-sdk/tree/develop/packages/service#get-an-access-token
 BTC_SERVICE_TOKEN=
 
-# The BTC assets api origin which should be matched with NETWORK and BTC_TESTNET_TYPE
-BTC_SERVICE_ORIGIN=https://btc-assets-api.testnet.mibao.pro
+# The BTC assets api origin which should match `BTC_SERVICE_TOKEN`
+# JTS Debugger: https://jwt.io
```
Comment (Flouse) [inline apps/service/README.md:47]:
```suggestion
# JWT Debugger: https://jwt.io
```

**Review by Flouse — APPROVED** [review]

**Pull request merged**